tools/runners/run-all.js

```diff
@@ -163,6 +178,8 @@ class Runner {
 
     if (! self.stopped) {
       buildmessage.enterJob({ title: "starting your app" }, function () {
+        self.errorAppRunner.start();
+        runLog.log('Started error app.', {arrow: true});
```

Comment:
I thought it'd be useful to know if the error app ever fails for whatever reason - instead of the regular main app they're trying to load.